diff -urNp -x '*.orig' QScintilla_gpl-2.11.2.org/Python/configure.py QScintilla_gpl-2.11.2/Python/configure.py
--- QScintilla_gpl-2.11.2.org/Python/configure.py	2021-03-05 20:11:27.686780829 +0100
+++ QScintilla_gpl-2.11.2/Python/configure.py	2021-03-05 20:11:27.783446925 +0100
@@ -1354,8 +1354,6 @@ def _generate_code(target_config, opts,
     sip_file = os.path.join(target_config.src_dir, module_config.get_sip_file(target_config))
 
     head, tail = os.path.split(sip_file)
-    while head:
-        head, tail = os.path.split(head)
 
     if tail != sip_file:
         argv.append('-I')
