diff -urNp -x '*.orig' QScintilla_gpl-2.11.2.org/Python/configure.py QScintilla_gpl-2.11.2/Python/configure.py
--- QScintilla_gpl-2.11.2.org/Python/configure.py	2021-03-05 20:11:27.933446039 +0100
+++ QScintilla_gpl-2.11.2/Python/configure.py	2021-03-05 20:11:28.030112136 +0100
@@ -645,7 +645,7 @@ class _HostPythonConfiguration:
 
         self.inc_dir = sysconfig.get_python_inc()
         self.venv_inc_dir = sysconfig.get_python_inc(prefix=sys.prefix)
-        self.module_dir = sysconfig.get_python_lib(plat_specific=1)
+        self.module_dir = os.path.join(sysconfig.get_python_lib(plat_specific=1,standard_lib=1), 'site-packages')
         self.debug = hasattr(sys, 'gettotalrefcount')
 
         if sys.platform == 'win32':
@@ -1564,7 +1564,10 @@ sip.files =''' % quote(path))
             fn = fn.replace(os.sep, '/')
             fn = os.path.normpath(fn)
 
-            pro.write(' \\\n    ../%s' % fn)
+            if os.path.isabs(fn):
+                pro.write(' \\\n    %s' % fn)
+            else:
+                pro.write(' \\\n    ../%s' % fn)
 
         pro.write('''
 INSTALLS += sip
