diff -urNp -x '*.orig' QScintilla_gpl-2.11.2.org/Python/configure.py QScintilla_gpl-2.11.2/Python/configure.py
--- QScintilla_gpl-2.11.2.org/Python/configure.py	2021-03-05 20:11:27.200117035 +0100
+++ QScintilla_gpl-2.11.2/Python/configure.py	2021-03-05 20:11:27.296783130 +0100
@@ -294,7 +294,7 @@ class ModuleConfiguration(object):
             return None
 
         path = os.path.join(target_configuration.qsci_sip_dir, 'Qsci')
-        files = glob.glob('sip/*.sip')
+        files = glob.glob(os.path.join(target_configuration.src_dir, 'sip/*.sip'))
 
         return path, files
 
@@ -1351,7 +1351,7 @@ def _generate_code(target_config, opts,
     # any '-I' needed appears first.
     pkg_root = os.path.dirname(os.path.abspath(__file__))
 
-    sip_file = module_config.get_sip_file(target_config)
+    sip_file = os.path.join(target_config.src_dir, module_config.get_sip_file(target_config))
 
     head, tail = os.path.split(sip_file)
     while head:
@@ -1823,8 +1823,11 @@ def _main(argv, pkg_config):
     arguments.  pkg_config is the package configuration.
     """
 
+    src_dir = os.path.dirname(os.path.abspath(__file__))
+
     # Create the default target configuration.
     target_config = _TargetConfiguration(pkg_config)
+    target_config.src_dir = src_dir
 
     # Parse the command line.
     p = _create_optparser(target_config, pkg_config)
