--- QScintilla-gpl-2.8.4/Qt4Qt5/qscintilla.pro.orig	2014-09-11 19:15:15.000000000 +0200
+++ QScintilla-gpl-2.8.4/Qt4Qt5/qscintilla.pro	2014-11-03 20:11:22.685915741 +0100
@@ -28,7 +28,7 @@
 !win32:VERSION = 11.3.0
 
 TEMPLATE = lib
-TARGET = qscintilla2
+TARGET = qscintilla2-qt$$QT_MAJOR_VERSION
 CONFIG += qt warn_off release thread exceptions
 INCLUDEPATH += . ../include ../lexlib ../src
 
--- QScintilla-gpl-2.8.4/Qt4Qt5/features/qscintilla2.prf.orig	2014-09-11 19:15:15.000000000 +0200
+++ QScintilla-gpl-2.8.4/Qt4Qt5/features/qscintilla2.prf	2014-11-03 20:35:52.655854645 +0100
@@ -6,20 +6,22 @@
     }
 }
 
+QTSUF = -qt$$QT_MAJOR_VERSION
+
 INCLUDEPATH += $$[QT_INSTALL_HEADERS]
 
 LIBS += -L$$[QT_INSTALL_LIBS]
 
 CONFIG(debug, debug|release) {
     mac: {
-        LIBS += -lqscintilla2_debug
+        LIBS += -lqscintilla2$${QTSUF}_debug
     } else {
         win32: {
-            LIBS += -lqscintilla2d
+            LIBS += -lqscintilla2$${QTSUF}d
         } else {
-            LIBS += -lqscintilla2
+            LIBS += -lqscintilla2$${QTSUF}
         }
     }
 } else {
-    LIBS += -lqscintilla2
+    LIBS += -lqscintilla2$${QTSUF}
 }
