diff -urNp -x '*.orig' QScintilla_gpl-2.11.2.org/Python/configure.py QScintilla_gpl-2.11.2/Python/configure.py
--- QScintilla_gpl-2.11.2.org/Python/configure.py	2021-03-05 20:11:27.443448932 +0100
+++ QScintilla_gpl-2.11.2/Python/configure.py	2021-03-05 20:11:27.536781714 +0100
@@ -306,7 +306,7 @@ class ModuleConfiguration(object):
         version of Qt.  target_configuration is the target configuration.
         """
 
-        qmake = {'CONFIG': 'qscintilla2'}
+        qmake = {'CONFIG': 'qscintilla2', 'Qt5:QT': 'widgets printsupport'}
 
         if target_configuration.qsci_inc_dir is not None:
             qmake['INCLUDEPATH'] = quote(target_configuration.qsci_inc_dir)
