--- QScintilla-2.8-gpl/designer-Qt4Qt5/designer.pro.orig	2006-12-25 04:56:07.000000000 +0100
+++ QScintilla-2.8-gpl/designer-Qt4Qt5/designer.pro	2006-12-26 02:54:57.792560260 +0100
@@ -3,6 +3,8 @@
 
 TEMPLATE = lib
 TARGET = qscintillaplugin
+INCLUDEPATH = ../Qt4Qt5
+LIBPATH = ../Qt4Qt5
 
 CONFIG += designer release plugin
 
--- QScintilla-gpl-2.8.4/Python/configure.py.orig	2014-09-11 19:15:17.000000000 +0200
+++ QScintilla-gpl-2.8.4/Python/configure.py	2014-11-02 14:41:09.187039766 +0100
@@ -1418,7 +1418,7 @@
 
     includepath = qmake_config.get('INCLUDEPATH')
     if includepath:
-        pro.write('INCLUDEPATH += %s\n' % includepath)
+        pro.write('INCLUDEPATH = %s $$INCLUDEPATH\n' % includepath)
 
     # Make sure the SIP include directory is searched before the Python include
     # directory if they are different.
