diff -urNp -x '*.orig' QScintilla_gpl-2.11.2.org/Python/configure.py QScintilla_gpl-2.11.2/Python/configure.py
--- QScintilla_gpl-2.11.2.org/Python/configure.py	2021-03-05 20:11:28.180111251 +0100
+++ QScintilla_gpl-2.11.2/Python/configure.py	2021-03-05 20:11:28.276777348 +0100
@@ -880,14 +880,14 @@ class _TargetConfiguration:
             try:
                 from PyQt5 import QtCore
             except ImportError:
-                error(
+                inform(
                         "Unable to import PyQt5.QtCore. Make sure PyQt5 is "
                         "installed.")
         else:
             try:
                 from PyQt4 import QtCore
             except ImportError:
-                error(
+                inform(
                         "Unable to import PyQt4.QtCore. Make sure PyQt4 is "
                         "installed.")
 
